tests/framework/CodeInterfaceTests/Neutrino/Neutrino_Base/out_dump.xml

```diff
@@ -0,0 +1,14 @@
+<DataObjectMetadata name="resultSet">

```

Comment:
you can delete this... 
It is not in the gold folder and you do not check for this